scripts/check-cocoapods-locks.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Early exit when script is run on platforms other than macOS
+if [[ "$OSTYPE" != "darwin"* ]]; then
+  exit 0
+fi
+
+green="tput setaf 2"
+yellow="tput setaf 3"
+blue="tput setaf 4"
+reset="tput sgr0"
+
+HEADER_PRINTED=false
+
+check-cocoapods-locks() {
+  podfileLockPath="$1/Podfile.lock"
+  manifestLockPath="$1/Pods/Manifest.lock"
+  podfileLock=`md5 -q $podfileLockPath 2>&1`
```

Comment:
prefer POSIX `$()` over backticks, send error text to /dev/null instead of capturing it
```suggestion
  podfileLock=$(md5 -q "$podfileLockPath" 2>/dev/null)
```